### DISCUSSION TITLE
How to add citations (followed the instructions but failed)

### DISCUSSION THREAD

**Discussion by XIYANG1018** [greenelab/lab-website-template#239] [Q&A] [answered]
Hi all!

I have a question about updating the citations. I have added the id (doi number) in sources.yaml file, but it seems that the cite process does not start? I am not sure if I missed any step.. Do I need to connect this source file with the cite process somewhere in the project? Btw, there was no error when I added a new doi, so it seems that it can read the doi but there was no output in citation file?

https://github.com/XIYANG1018/pain-lab-website

I tried to run docker but it didn't send me a localhost link so I user bundle exec jekyll serve to preview the page. This is the error message: [FATAL tini (7)] exec /var/entrypoint.sh failed: No such file or directory

**Comment by XIYANG1018** [comment, marked as answer]
I've tried another solution which is not to use docker and made it!!

**Reply by vincerubinetti** [reply]
Can you be more specific with details on this? I would really like to know what went wrong here.

For example, what operating system and version were you trying to run Docker on? Are you sure the Docker process or daemon was running? What exactly was all of the console output? Which parts of the docs did you follow, what commands did you run? How far in the [Docker steps](https://greene-lab.gitbook.io/lab-website-template-docs/getting-started/preview-your-site#on-your-computer-locally) did you get? Would you be open to us giving you some debug commands to run?

Maybe something from this issue you had below had inconspicuously stuck around to cause this issue.
https://github.com/greenelab/lab-website-template/discussions/233

**Reply by XIYANG1018** [reply]
I was stuck at the second step when running ./.docker/run.sh. I got the console output as below.
![image](https://github.com/greenelab/lab-website-template/assets/117253597/11271047-1c77-426d-8677-7c30da7cc5b5)

I am running Docker on Windows 11. There was a time I got the localhost page link on my console but the page didn't show the images, all images became black background so I then change to the bundle jekyll solution.

**Reply by vincerubinetti** [reply]
For now I'm going to mark this as answered, because:

- You found a work around.
- I'd need a lot more details to really be able to diagnose this, possibly a live screensharing session.
- I'm guessing and hoping that this issue was caused by [your previous issue here](https://github.com/greenelab/lab-website-template/discussions/233).
- The images becoming black can't have anything to do with the docker really, that happens all in the browser. I don't know what could cause that other than some bad CSS, the images actually being black, or there being something wrong with your browser or system.
